scripts/bootstrap.sh

```diff
@@ -7,6 +7,7 @@ echo "core:core" | chpasswd
 
 systemctl enable docker.service
 systemctl enable kubelet.service
+systemctl enable rktlet.service
```

Comment:
Does it mean that we run rktlet unconditionally, even if we are using docker runtine? Or am I missing something?